spec.bs

```diff
@@ -18,3 +18,122 @@ Default Biblio Status: current
 Markup Shorthands: markdown yes
 Assume Explicit For: yes
 </pre>
+
+
+<h2 id="advertisements">Advertisements</h2>
+
+An {{AuctionAd}} contains the following:
+
+* A <dfn for=AuctionAd dfn-type="dict-member">renderUrl</dfn> .
+* An optional <dfn for=AuctionAd dfn-type="dict-member">metadata</dfn> which is text containing opaque JSON encoded data.
+
+<xmp class="idl">
+dictionary AuctionAd {
+  required USVString renderUrl;
+  any metadata;
+};
+</xmp>
+
+
+<h2 id="advertisement-interest-groups">Advertisement Interest Groups</h2>
+
+An <dfn export>advertisement interest group</dfn>
+is a [=dictionary=] with the following:
+ 
+* An <dfn export for="AuctionAdInterestGroup">expiration</dfn> point in time.
+
+* An <dfn export for="AuctionAdInterestGroup">owner</dfn> [=origin=].
+
+* A <dfn export for="AuctionAdInterestGroup">name</dfn>, a [=string=].
+ 
+* A <dfn export for="AuctionAdInterestGroup">priority</dfn>, a {{double}}, initially 0.
+ 
+* An <dfn export for="AuctionAdInterestGroup">enableBiddingSignalsPrioritization</dfn>, a [=boolean=], initially false.
+ 
+* An optional <dfn export for="AuctionAdInterestGroup">priorityVector</dfn>, A [=map=] from [=string=] to {{double}}.
+ 
+* An optional <dfn export for="AuctionAdInterestGroup">prioritySignalsOverrides</dfn>, A [=map=] from [=string=] to {{double}}.
+ 
+* An <dfn export for="AuctionAdInterestGroup">execution_mode</dfn>, a {{WorkletExecutionMode}}.
+ 
+* An optional <dfn export for="AuctionAdInterestGroup">biddingLogicURL</dfn>, a [=url=].
+
+* An optional <dfn export for="AuctionAdInterestGroup">biddingWasmHelperUrl</dfn>, a [=url=].
+ 
+* An optional <dfn export for="AuctionAdInterestGroup">dailyUpdateUrl</dfn>, a [=url=].
+
+* An optional <dfn export for="AuctionAdInterestGroup">trustedBiddingSignalsUrl</dfn>, a [=url=].
+
+* A <dfn export for="AuctionAdInterestGroup">trustedBiddingSignalsKeys</dfn>, a [=list=] of [=strings=].
+ 
+* An optional <dfn export for="AuctionAdInterestGroup">userBiddingSignals</dfn>, a [=string=].
+ 
+* An <dfn export for="AuctionAdInterestGroup">ads</dfn>, a [=list=] of {{AuctionAd}}.

```

Comment:
Should this be pluralized? like "a list of AuctionAds"?

Reply:
Doesn't work.  Maybe there is a notation for linking from "AuctionAds" to the AuctionAd definition.